### DISCUSSION TITLE
Can schedules have a duration of the schedule?

### DISCUSSION THREAD

**Discussion by adamjk-dev** [chaos-mesh/chaos-mesh#2486] [Q&A]
I saw durations for the chaos event types in a schedule. But, imagine that I want to run a PodChaos/PodKill event every minute, for a total duration of say, 2 hours.

I want the schedule to run for a period of time, then stop and cleanup on its own, per say. I didn't see that, I saw a duration for the "event" but not a time period to let a schedule run in total.

Any help would be appreciated. Thanks!

**Comment by alezar** [comment]
While solving similar task I used crontab to limit duration period by providing exact hours and date.
Schedule itself does not stop and cleanup, but at least I am able to run a scenario without the need of manually pause it.

For example I used 
*/15 12,13 01 12 * 
to run PodKill every 15 minutes for 2 particular hours at a particular date.

Hope that helps!

**Reply by adamjk-dev** [reply]
Makes sense, I was more or less looking for a max duration of time. For instance, I want a schedule to run for a maximum of X hours and then be removed.

**Reply by Andrewmatilde** [reply]
I think the problem is the workflow now do not have ability to  loop executing.
